core-bundle/src/DependencyInjection/Attribute/AsPage.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of Contao.
+ *
+ * (c) Leo Feyer
+ *
+ * @license LGPL-3.0-or-later
+ */
+
+namespace Contao\CoreBundle\DependencyInjection\Attribute;
+
+/**
+ * An attribute class for page controllers.
+ *
+ * @see \Symfony\Component\Routing\Annotation\Route
+ */
+class AsPage
```

Comment:
No `#[\Attribute(\Attribute::TARGET_CLASS | \Attribute::IS_REPEATABLE)]` here?

Reply:
fixed in 44c5d82c75a9f8fdeaaef8760e97c51f702f34f5